draft-wiggers-hbs-state.md

```diff
@@ -94,6 +94,14 @@ informative:
     date: October 2020
     seriesinfo: NIST Special Publication
 
+  FIPS204:
+    target: https://doi.org/10.6028/NIST.FIPS.204
+    title: "FIPS 204: Module-Lattice-Based Digital Signature Standard"
+    author:
+      - ins: National Institute of Standards and Technology
+    date: 2024-08-13
+    seriesinfo: Federal Information Processing Standards
+
   FIPS205:
     target: https://doi.org/10.6028/NIST.FIPS.205
     title: "FIPS 205: Stateless Hash-Based Digital Signature Standard"
```

Comment:
nit: Let's add the author/date/series info to FIPS205 well.

Reply:
The FIPS204 section has been copy-pasted, so it's there already below the fold.